rskj-core/src/main/java/co/rsk/peg/constants/BridgeRegTestConstants.java

```diff
@@ -53,12 +50,9 @@ public class BridgeRegTestConstants extends BridgeConstants {
     public BridgeRegTestConstants(List<BtcECKey> federationPublicKeys) {
         btcParamsString = NetworkParameters.ID_REGTEST;
 
-        List<FederationMember> federationMembers = FederationMember.getFederationMembersFromKeys(federationPublicKeys);
-
-        Instant genesisFederationCreatedAt = ZonedDateTime.parse("2016-01-01T00:00:00Z").toInstant();
+        this.genesisFederationPublicKeys = federationPublicKeys;
 
-        FederationArgs federationArgs = new FederationArgs(federationMembers, genesisFederationCreatedAt, 1L, getBtcParams());
-        genesisFederation = FederationFactory.buildStandardMultiSigFederation(federationArgs);
+        genesisFederationAddressCreatedAt = ZonedDateTime.parse("2016-01-01T00:00:00Z").toInstant();
```

Comment:
I like how this looks, rather than how it's done in other constant classes that has the milliseconds value hardcoded and a comment with the date.

Do you think we could apply this in the other constants files? Make sure you get the same result

Reply:
Done! Here is some evidence where we made sure is keeping the same value related to the milliseconds previously expressed:


<img width="1022" alt="image" src="https://github.com/rsksmart/rskj/assets/160710714/56dd9c72-941b-4cef-b61a-d4696b7cbcd6">

<img width="1003" alt="image" src="https://github.com/rsksmart/rskj/assets/160710714/6dd717d3-397a-4ae3-a59e-55608faf8e29">

<img width="990" alt="image" src="https://github.com/rsksmart/rskj/assets/160710714/bf449fcd-51d2-4c0b-87c3-6c659b8ca55b">

Reply:
Better to add a unit test :)